src/components/Header/Header.module.scss

```diff
@@ -158,7 +158,7 @@
   align-self: center;
 
   svg {
-    fill: var(--color-secondary);
+    fill: currentColor;
```

Comment:
svg still has style attribute, so when the colors are set properly, it should work out of box

<img width="576" alt="Screenshot 2023-01-30 at 14 51 38" src="https://user-images.githubusercontent.com/91472483/215495674-fac87503-5d08-47a0-b5cc-a5efdef59c41.png">